packages/addons/addon-depends/dotnet-runtime-depends/aspnet5-runtime/package.mk

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="aspnet5-runtime"
+PKG_VERSION="5.0.7"
+PKG_ARCH="any"
+PKG_LICENSE="MIT"
+PKG_SITE="https://dotnet.microsoft.com/"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="tools"
+PKG_SHORTDESC="ASP.NET Core Runtime"
+PKG_LONGDESC="ASP.NET Core Runtime (${PKG_VERSION}) enables you to run existing web/server applications."
+PKG_TOOLCHAIN="manual"
+
+case "${ARCH}" in
+  "aarch64")
+    PKG_SHA256=""
+    PKG_URL="https://download.visualstudio.microsoft.com/download/pr/d0a22fa3-b916-49ce-8284-97131b424cb3/cb884163ad34b83f1ae1dbd33e09d77a/aspnetcore-runtime-5.0.7-linux-arm64.tar.gz"
```

Comment:
5.0.7 hardcoded ?
I guess the whole url changes at every version ?

Reply:
correct - same as 3.x :-(